emr-user-role-mapper-application/src/main/java/com/amazon/aws/emr/ApplicationConfiguration.java

```diff
@@ -100,6 +100,12 @@ public void setProperty(String propertyName, String value) {
     public Set<String> getAllowedUsersForImpersonation() {
         return this.IMPERSONATION_ALLOWED_USERS;
     }
+
+    public boolean isSetSourceIdentityEnabled() {
+        return Boolean.parseBoolean(properties.getProperty(Constants.SET_SOURCE_IDENTITY_ENABLED,
```

Comment:
What is the drawback of having this enabled by default?

Reply:
When would someone NOT want to have this enabled is probably a better question. I think we should always set it.